app/src/main/java/org/dhis2/data/forms/dataentry/DataEntryAdapter.kt

```diff
@@ -25,7 +25,14 @@ class DataEntryAdapter(private val searchStyle: Boolean) :
         R.layout.form_scan,
         R.layout.form_org_unit,
         R.layout.form_coordinate_custom,
-        R.layout.form_edit_text_custom
+        R.layout.form_edit_text_custom,
+        R.layout.form_option_set_selector,
+        R.layout.form_option_set_spinner,
+        R.layout.form_unsupported,
```

Comment:
This view was removed in another PR, and now that PR is merged in the base branch